### ISSUE TITLE
1 comment obscured by comments label

### ISSUE THREAD

**Issue by mrchristian** [fiduswriter/fiduswriter#1271]
If a user comments on a Document title then the first part of the comment, along with the name of the person making the comment. It would be helpful if the first comment start height was lowered a little.

See screenshot:

![Fidus-Writer](https://github.com/fiduswriter/fiduswriter/assets/772579/5d3484cb-96c0-4c17-8d48-89d8aaa9c6de)

**Comment by nmueller18** [comment]
I agree. This also relates to tracked changes in the title. To reach those, the window has to be adjusted until the mobile settings kick in so that comments are displayed at the bottom of the window.
